python-avocado.spec

```diff
@@ -567,7 +569,7 @@ server.
 
 %package -n python2-%{srcname}-plugins-varianter-yaml-to-mux
 Summary: Avocado plugin to generate variants out of yaml files
-Requires: %{name} == %{version}
+Requires: python2-%{srcname} == %{version}
```

Comment:
This is indeed a nice to have, but shouldn't have been necessary, because:

```
$ rpm -qp --provides python2-avocado-60.0-2.fc27.noarch.rpm 
warning: python2-avocado-60.0-2.fc27.noarch.rpm: Header V4 RSA/SHA256 Signature, key ID a5f209f3: NOKEY
python-avocado = 60.0-2.fc27
python2-avocado = 60.0-2.fc27
```

The Python 3 requires, though, are absolute necessary! Thanks for spotting those.